packages/appstore/src/components/containers/trading-app-card.tsx

```diff
@@ -50,6 +54,26 @@ const TradingAppCard = ({
     );
 
     const openStaticPage = () => {
+        if (is_deriv_platform) {
+            switch (name) {
+                case DERIV_PLATFORM_NAMES.TRADER:
+                    window.open(getStaticUrl(`/dtrader`));
+                    break;
+                case DERIV_PLATFORM_NAMES.DBOT:
+                    window.open(getStaticUrl(`/dbot`));
+                    break;
+                case DERIV_PLATFORM_NAMES.SMARTTRADER:
+                    window.open(getUrlSmartTrader());
+                    break;
+                case DERIV_PLATFORM_NAMES.BBOT:
+                    window.open(getUrlBinaryBot());
```

Comment:
@heorhi-deriv this links goes to [staging-bot](https://staging-bot.deriv.com/), not to [bot](https://bot.deriv.com/)

Reply:
hey @sergei-deriv yeap this is intended. if we check `getUrlBinaryBot` function, there are paths that we check:
```
  const resolutions = {
        is_staging_deriv_app: /^staging-app\.deriv\.(com|me|be)$/i.test(domain),
        is_deriv_app: /^app\.deriv\.(com|me|be)$/i.test(domain),
        is_test_link: /^(.*)\.binary\.sx$/i.test(domain),
    };
```
so test links, and localhost will redirect you to staging. like if you click on the Open button in trading app card.

Reply:
@sergei-deriv seems like we should improve it, checking with Rupato regarding that, thanks for notice